plugins/com.google.cloud.tools.eclipse.appengine.standard.java8/plugin.xml

```diff
@@ -86,38 +86,36 @@
   <extension point="com.google.cloud.tools.eclipse.appengine.libraries"
              id="com.google.cloud.tools.eclipse.appengine.standard.java8.apis">   
     <library
-          id="servlet-api-3.1"
-          name="Servlet API 3.1"
+          id="appengine-servlet-3.1"
+          name="App Engine: Servlet API 3.1"
           group="servlet"
           siteUri="http://www.oracle.com/technetwork/java/javaee/servlet/index.html"
           export="false"
           recommendation="required">
-    
+      <libraryDepency id="appengine-api"/>
```

Comment:
libraryDependency (and suggests a test is missing)

Reply:
I was thinking that after I packed up last night: it would be good to have integration tests that verified JSP, JSTL, EL, and the App Engine SDK.

Reply:
typo still not fixed